ntc_templates/templates/huawei_vrp_display_bgp_peer.textfsm

```diff
@@ -0,0 +1,20 @@
+Value PEER ([\w\.:]+)
+Value V (\d+)
+Value AS ([\d\.]+)
+Value MSG_RCV (\d+)
+Value MSG_SENT (\d+)
+Value OUT_Q (\d+)
+Value UP_DOWN ([\w:]+)
+Value STATE (\w+)
+Value PREF_RCV (\d+)
+
+Start
+  ^\s+${PEER}\s+${V}\s+${AS}\s+${MSG_RCV}\s+${MSG_SENT}\s+${OUT_Q}\s+${UP_DOWN}\s+${STATE}\s+${PREF_RCV} -> Record
+  ^\s+Status codes
+  ^\s+BGP local router ID
+  ^\s+Local AS number
+  ^\s+Total number of peers
+  ^\s+Peers in established state
+  ^\s+Total number of dynamic peers
```

Comment:
Let's convert literal white spaces to regexes.

To accept my suggestion, please click the "commit suggestion" button. Thank you!

```suggestion
  ^\s+Status\s+codes
  ^\s+BGP\s+local\s+router ID
  ^\s+Local\s+AS\s+number
  ^\s+Total\s+number\s+of\s+peers
  ^\s+Peers\s+in\s+established\s+state
  ^\s+Total\s+number\s+of\s+dynamic\s+peers
```